symplyphysics/laws/optics/optical_power_from_thin_lens_radius.py

```diff
@@ -0,0 +1,52 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, dimensionless, validate_input,
+    validate_output)
+
+# Description
+## Optical power is the degree to which a lens, mirror, or other optical system converges or diverges light.
+## For two or more thin lenses close together, the optical power of the combined lenses is approximately
+## equal to the sum of the optical powers of each lens.
+
+## Law: P = (n - n0)(1/R1 - 1/R2)
+## Where:
+## P - optical power of thin lense,
+## n - refractive index of lens material,
+## n0 - refractive index of medium,
+## R1 - radius of the front surface of lens,
+## R2 - radius of the back surface of lens.
+
+# Conditions
+## It is important to consider the sign of the radius of the surface!
```

Comment:
Put verbose explanation in description and short in conditions. Short looks like: R > 0 for convex surface, R < 0 for concave.